code/jvm/secrets/README.md

```diff
@@ -0,0 +1,2 @@
+This folder contains sensitive information such as the GitHub App's private key and the cipher key.
```

Comment:
- Where and how are they replaced?
- There shouldn't be a static location in the source for these file.

Reply:
In the future, this information will probably be stored in environment variables, as in, the path for the .PEM file, and the key to use in the cipher.